tendermint/src/validator.rs

```diff
@@ -192,6 +192,7 @@ pub struct Update {
     pub pub_key: PublicKey,
 
     /// New voting power
+    #[serde(default)]
```

Comment:
Ouch, thanks! I'm wondering if there shouldn't be two test cases (one with power and one without) instead of one. This is all so brittle :-/ We do not test all edge cases well. ref #303